sdfObject/sdfobject-genericlevel.sdf.json

```diff
@@ -0,0 +1,132 @@
+            },
+            "RemainingTime": {
+              "sdfRef": "#/sdfObject/GenericLevel/sdfData/TransitionTimeData"
+            }
+          }
+        },
+        "LevelSet": {
+          "description": "Set the level server state with time and delay parameters",
+          "sdfInputData": [
+            "#/sdfObject/GenericLevel/sdfAction/LevelSet/sdfData/Level",
+            "#/sdfObject/GenericLevel/sdfAction/LevelSet/sdfData/TransitionTime",
+            "#/sdfObject/GenericLevel/sdfAction/LevelSet/sdfData/Delay"
+          ],
+          "sdfData": {
+            "Level": {
+                "sdfRef": "#/sdfObject/GenericLevel/sdfData/GenericLevelData"
+            },
+            "TransitionTime": {
+              "sdfRef": "#/sdfObject/GenericLevel/sdfData/TransitionTimeData"
+            },
+            "Delay": {
+              "sdfRef": "#/sdfObject/GenericLevel/sdfData/DelayData"
+            }
+          }
+        },
+        "DeltaSet": {
+          "description": "Set the level server state with time and delay parameters",
+          "sdfInputData": [
+            "#/sdfObject/GenericLevel/sdfAction/DeltaSet/sdfData/DeltaLevel",
+            "#/sdfObject/GenericLevel/sdfAction/DeltaSet/sdfData/TransitionTime",
+            "#/sdfObject/GenericLevel/sdfAction/DeltaSet/sdfData/Delay"
+          ],
+          "sdfData": {
+            "DeltaLevel": {
+              "sdfRef": "#/sdfObject/GenericLevel/sdfData/GenericLevelData"
+            },
+            "TransitionTime": {
+              "sdfRef": "#/sdfObject/GenericLevel/sdfData/TransitionTimeData"
+            },
+            "Delay": {
+              "sdfRef": "#/sdfObject/GenericLevel/sdfData/DelayData"
+            }
+          }
+        },
+        "MoveSet": {
+          "description": "Set the level server state with time and delay parameters",
+            "sdfInputData": [
+              "#/sdfObject/GenericLevel/sdfAction/MoveSet/sdfData/DeltaLevel",
+              "#/sdfObject/GenericLevel/sdfAction/MoveSet/sdfData/TransitionTime",
+              "#/sdfObject/GenericLevel/sdfAction/MoveSet/sdfData/Delay"
+            ],
+            "sdfData": {
+            "Level": {
+              "sdfRef": "#/sdfObject/GenericLevel/sdfData/GenericLevelData"
+            },
+            "TransitionTime": {
+              "sdfRef": "#/sdfObject/GenericLevel/sdfData/TransitionTimeData"
+            },
+            "Delay": {
+              "sdfRef": "#/sdfObject/GenericLevel/sdfData/DelayData"
+            }
+          }
+        }
+      },
+      "sdfData": {
+        "GenericLevelData": {
+          "description": "level state data",
+          "type": "number",
+          "minimum": -32768,
+          "maximum": 32767,
+          "multipleOf": 1
+        },
+        "TransitionTimeData": {
+          "description": "Generic Transition Time Format",
+          "type": "array",
+          "items": [
```

Comment:
this looks like TransitionTimeData : [ 100.0, "100ms"] which is a mixed array with floats & strings, which we do not support.
work around would be [ { 100.0, "100mS"}] but that is an array with JSON objects, which we do not support either

what would work is
TransitionTime Data =  [  100.0, 5.0 ]
where 5 can be the actual ms step resolution.
we can define that the array would always a multiple of 2.
(not i would prefer the json object).